hw1/TasksForHomework/Task1.java

```diff
@@ -0,0 +1,28 @@
+//package TasksForHomework;
+
+import java.util.Scanner;
+
+public class Task1 {
+
+    public static void main(String[] args) {
+
+        Scanner s1 = new Scanner(System.in);
+        System.out.println("Enter ur dogs age(years):");
+        int ageYears = s1.nextInt();
+
+        Scanner s2 = new Scanner(System.in);
+        System.out.println("Enter ur dogs age(months):");
+        int ageMonths = s2.nextInt();
+
+        Scanner s3 = new Scanner(System.in);

```

Comment:
you can create scanner only 1 time and then use it, instead of creating it 3 times